conf/application.conf

```diff
@@ -354,12 +354,12 @@ health {
 
   nsi-connection {
```

Comment:
Pls add a comment here
// MDTP platform default timeout is 60s, NS&I JBoss/FUSE is 40s, NS&I Thaler db is 29s
// The health check timeout will therefore be set between the first two, i.e. 50s to allow the timeout source to be identified.